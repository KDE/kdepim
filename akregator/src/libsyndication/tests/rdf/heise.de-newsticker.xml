<?xml version="1.0" encoding="utf-8"?>
<!-- real life RDF 0.9 feed -->
<rdf:RDF
xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
xmlns="http://my.netscape.com/rdf/simple/0.9/">

<channel>
  <title>heise online News</title>
  <link>http://www.heise.de/newsticker/</link>
  <description>Nachrichten nicht nur aus der Welt der Computer</description>
</channel>


<item>
  <title>Kreditkarten-Software soll unerlaubte Datenspeicherung ermöglichen</title>
  <link>http://www.heise.de/newsticker/meldung/71009</link>
</item>

<item>
  <title>Branche diskutiert über Chip-Testzentrum in Dresden</title>
  <link>http://www.heise.de/newsticker/meldung/71010</link>
</item>


</rdf:RDF>
