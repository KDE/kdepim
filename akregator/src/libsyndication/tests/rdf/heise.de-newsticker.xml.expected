# Feed begin ######################
title: #heise online News#
link: #http://www.heise.de/newsticker/#
description: #Nachrichten nicht nur aus der Welt der Computer#
# Item begin ######################
id: #hash:72556a424db41b9ee0176282b52bb8b6#
title: #Kreditkarten-Software soll unerlaubte Datenspeicherung ermöglichen#
link: #http://www.heise.de/newsticker/meldung/71009#
# Item end ########################
# Item begin ######################
id: #hash:713f6d48ba9d6d469ed72d425920447c#
title: #Branche diskutiert über Chip-Testzentrum in Dresden#
link: #http://www.heise.de/newsticker/meldung/71010#
# Item end ########################
# Feed end ########################

